### DISCUSSION TITLE
Luxon Settings in Angular apps

### DISCUSSION THREAD

**Discussion by awdorrin** [moment/luxon#1472] [Q&A]
Hello, we just started migrating our application from using a mix of JS Date and moment to luxon and it looks like the luxon API is going to help us resolve several issues we've been encountering with converting local time zones properly.

So far, in every Angular component's constructor, we start off with: `Settings.defaultZone = "America/New_York";`

What I am wondering is if this is the correct approach, or if there is some other way we can configure Settings globally within an Angular application. I spent a fair amount of time searching and haven't found anything yet and feel like I am probably missing something incredibly obvious to do this globally, or at least at a module level, rather than comoponent/service/etc.

Thanks!

**Comment by diesieben07** [comment]
Unless you are importing Luxon multiple times in your applications and therefor end up with multiple `Settings` classes (e.g. due to code splitting), it is enough to set this once, e.g. in `main.ts` or your root Angular component or module. `Settings.defaultZone` is a static property and therefor a global value.
